src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerImageReference.cs

```diff
@@ -65,27 +65,5 @@ public override object Id
                 return _id;
             }
         }
-
-        private string GetDebuggerDisplay()
```

Comment:
@zebmason This particular one is being used: the DebuggerDisplay attribute on the class will make this one matter.